### PR TITLE
Fix typos for "added" and "called"

### PR DESCRIPTION
**Pull request by pwoznic**
The words "added" and "called" were mispelled so I corrected them in the Custom Dimensions topic

### CODE REVIEW

File: www/docs/common-use-cases/semantic-search/custom_dimensions.md [hunk 1]
```diff
@@ -95,10 +95,10 @@ END
 In order to calculate the final score of a document and query that contains
 custom dimensions, <Config v="names.product"/> takes the dot product of the
 query's custom dimensions with the document's custom dimensions and the resulting
-number is aded to their score.
+number is added to their score.
 
-Negative values decrease the overall score (sometimes caled "burying") and
-postive values increase the overal score (sometimes caled "boosting").  A dot
+Negative values decrease the overall score (sometimes called "burying") and
+postive values increase the overal score (sometimes called "boosting").  A dot
```
Comment (eskibars) [inline www/docs/common-use-cases/semantic-search/custom_dimensions.md:101]: `overal` here should be updated to `overall`

FYI, the reason I made these typos before was that I was editing through my tablet which has some quirky behavior when it has repeated keys (2 `l`s in a row, e.g.). There might be more of these types of typos

**Review by eskibars — CHANGES_REQUESTED** [review]
One more change.  Thanks for the review/edit!

**Review by pwoznic — COMMENTED** [review]
Done.

**Review by pwoznic — COMMENTED** [review]
Updated other typos for overall, too, and the in this topic.

**Review by eskibars — APPROVED** [review]
LGTM, thanks!

**Pull request merged**